truetype/truetype.go

```diff
@@ -26,6 +26,32 @@ import (
 // An Index is a Font's index of a rune.
 type Index uint16
 
+// A NameID represents the Name Identifiers in the name table.
+type NameID uint16
+
+const (
+	NameIDCopyright NameID = iota
```

Comment:
I'd rather be explicit:
NameIDCopyright NameID = 0
NameIDFontFamily NameID = 1
etc
as the values are fixed in the file format, and I don't want adding or deleting constants here to change their values.

Below, delete NameIDReserved.